.github/workflows/build_cmake.yml

```diff
@@ -25,13 +25,17 @@ jobs:
             name: "Windows Build", artifact: "Windows.tar.xz",
             os: windows-latest,
             build_type: "Release", cc: "gcc-9", cxx: "g++-9",
-            cmake_platform: "-A x64"
```

Comment:
Note: `cmake_platform` is another os-specific cmake param (to configure instead of build), so I'm using the same technique for number_of_jobs.

Reply:
@gregstoll Charlie is out this week and this is failing ~50%. Can you sanity check buddy this and I'll check back with Charlie next week.